dev-util/byacc/byacc-20180510.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Berkeley implementation of Yacc"
+DESCRIPTION="Berkeley Yacc (byacc) is generally conceded to be the best yacc \
+variant available. In contrast to bison, it is written to avoid dependencies \
+upon a particular compiler."
+HOMEPAGE="http://invisible-island.net/byacc/byacc.html"
+COPYRIGHT="2002-2013,2018 by Thomas E. Dickey"
```

Comment:
The homepage shows: `Copyright © 2002-2017,2018 by Thomas E. Dickey` so I guess it would be OK to "join" `2002-2017` and `2018`:
```diff
-COPYRIGHT="2002-2013,2018 by Thomas E. Dickey"
+COPYRIGHT="2002-2018 Thomas E. Dickey"
```

Reply:
Wasn't sure about it :)